resources/frontend_client/inline_js/init.js

```diff
@@ -1,65 +1,69 @@
-var content =
-  [
-    ['drill-through-img', 'Click on your charts to dive deeper.'],
-    ['metabot-img shadow', 'Bring your charts and data into Slack.'],
-    ['calendar-img', 'Dashboard filters let you filter all your charts at once.'],
-    ['charts-img shadow', 'Easily create and share beautiful dashboards.'],
-    ['column-heading-img', 'Click on column headings in your tables to explore them.']
-  ];
+const content = [
```

Comment:
Every image has its own message and previously image name and the message were connected in this array because their meaning connected. Now we have gifs with nondescriptive names and connection between messages and images based on indices. This change seems like a complication of things, is there reasons for that?

Reply:
Thanks, reverted.